### ISSUE TITLE
Compile program assessment responses

### ISSUE THREAD

**Issue by brymz** [TREC-Agroecology/introducing-agroecology#5]

**Comment by Danielcalza** [comment]
Questions:

1.	Did you learn anything?

2.	Recommend to a friend?

3.	How will you incorporate AE into extension programs? 

4.	Better next time?

5.	Favorite part? 

Written responses:

1.	– History of, and extension perspective. Experiences from fellow colleagues, and what is happening in Florida. – Learned a lot. 

2.	– Yes, more people should take this course. – Especially livestock, residential horticulture, Master gardener, coordinator. 

3.	 – School and community gardens. Improved diversity and aesthetics.  – Developing agroecology international field trips i.e. Cuba.  – Offer training to new/ beginning farmers. Develop permaculture factsheet.  – Formulating ways to integrate, and already using some in other areas. – Help farmers when establishing new crops and acreage and combating existing challenges. – Already do some, plan to do more and use the term. – Will be adding to the existing program.  – Feeling more confident about the research-based fundamentals of AE. Will develop a CFL small farmer Network for idea-sharing. – Add to small farmer program and incorporate into homestead manual currently being worked on. – Integrating the concepts into a food systems approach. – Will develop a new program. – Add to existing program and help develop an EDIS document for permaculture. 

4.	– More interactive portions. Day 1: good interactivity. Day 2: more interactivity needed. – Economic dimension and present a case study. – Provide an agenda for the course. – Longer and/or more in-depth. – More water available for field portion. (2) – Have summer field trip in the AM instead. – 2 full-day course. – More examples of successful farmers apply AE concepts. – Field trip in the afternoon. – How to design an agroecological system. – Day 1 skipped around on media, hard to follow at times. 

5.	– Interactive portions and learning of the topic agroecology. – The farm visit was great. – Excellent speakers and location. – Farm tour and interacting with colleagues. – The drawing portions. – The group effect, good people, new colleagues. – The field trip was great. – Success stories i.e. working cattle ranch. – Learning about the Archbold research and realizing there is so much more to learn. – Good blend of content, experience, and discussion, and how other agents are already applying the concepts in their programs. – Case studies and tour. – Field trip and learning new ways of thinking through the concepts.  

Score totals:
1.	 4, 5, 5, 5, 4, 4, 5, 5, 5, 5, 5, 5: 57
2.	 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5: 60

**Comment by Danielcalza** [comment]
Program assessment responses noted and added.
